bookshelves.md

```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
 
 “Bookshelves” are lists of PG books that help users navigate PG. See them [here](https://www.gutenberg.org/wiki/Category:Bookshelf).
 
-
 ## Problem
 
 - System for creating and updating bookshelves is failing
-    - PHP/wiki interface
-    - Loaded into production Postgres db by scraping wiki!
-    - Few people maintaining it.
-    - No way to create personal bookshelves or reading lists
-    - Recurring security issues with PHP
-
+  - PHP/wiki interface
+  - Loaded into production Postgres db by scraping wiki!
+  - Few people maintaining it.
+  - No way to create personal bookshelves or reading lists
+  - Recurring security issues with PHP
 
 ## Proposed Solution
 
 - Registrationless Bookshelf Builder
-    - HTML5 local storage + javascript
-    - Button on Item page
-    - Import/export lists for sharing
-
-- Bookshelf Management App
-    - Python/Flask or Python/CherryPy with Postgres backend
-    - Managers can make new bookshelves and categories
-    - Moderators can add/remove items from Builder Bookshelves
 
+  - HTML5 local storage + javascript
+  - Button on Item page
+  - Import/export lists for sharing
 
+- Bookshelf Management App
+  - Python/Flask or Python/CherryPy with Postgres backend
+  - Managers can make new bookshelves and categories
+  - Moderators can add/remove items from Builder Bookshelves
 
 ## Goal
 
 Have prototype system running by April, 2021 for public review, move to production bin May.
 
+## Timeline
+
+| Dates               |                                                               Milestones                                                               |
+| ------------------- | :------------------------------------------------------------------------------------------------------------------------------------: |
+| 09/14/20 - 9/21/20  | Review Technical Specs, existing code/documentation, and implementation goals for deeper understanding of technical aspects of problem |
```

Comment:
What technical specs do you mean?

Reply:
I would suggest adding a week to document requirements of PG deployment environment and evaluate technology options.